index.js

```diff
@@ -21,11 +23,13 @@ function createRavenMiddleware(Raven, options = {}) {
     return next => action => {
       // Log the action taken to Raven so that we have narrative context in our
       // error report.
-      Raven.captureBreadcrumb({
-        category: breadcrumbCategory,
-        message: action.type,
-        data: breadcrumbDataFromAction(action)
-      });
+      if (filterBreadcrumbActions(action) === true) {
```

Comment:
I think it would be less confusing for this to just check for truthy values.

```
if (filterBreadcrumbActions(action)) {
```

Reply:
Yeah, I did that at first, but thought being explicit might catch some bugs in the user-supplied function.

Reply:
I think truths/falsy is a perfectly valid return value from a filter predicate. Not nessesarily a bug.